<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inputX</class>
 <widget class="QDialog" name="inputX">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>260</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>210</width>
    <height>260</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>210</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Enter X value</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
  margin: 0 0 5px;
  padding: 0 2px;
  font-size: 16px;
  font-weight: bold;
  color: #101010;
  background: #dfdfdf;
  border-width: 1px 1px 1px;
  border-style: solid;
  border-color:   #cecece #bababa #a8a8a8;
  border-radius: 13px 13px 12px 12px;
}

QPushButton::hover{
  background: #d5d5d5;
  border-top-color: #c9c9c9;
}

QPushButton::pressed{
  vertical-align: -5px;
  background: #c2c2c2;
  margin-bottom: 0;
  padding: 1px 13px 0;
  border-width: 0;
  border-radius: 15px;
}

QDialog{
  background: #eeeeee;
}

QLineEdit{
  background: #ffffff;
  color: #131313;
}</string>
  </property>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono NL</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>7</string>
   </property>
   <property name="shortcut">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono NL</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="shortcut">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono NL</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>9</string>
   </property>
   <property name="shortcut">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono NL</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
   <property name="shortcut">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono NL</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="shortcut">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono NL</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="shortcut">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono NL</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono NL</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono NL</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sign">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono NL</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Â±</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_dot">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono NL</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>.</string>
   </property>
   <property name="shortcut">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_0">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono NL</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="shortcut">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_enter">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono NL</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
  margin: 0 0 5px;
  padding: 0 12px;
  font-size: 16px;
  font-weight: bold;
  color: #eeeeee;
  background: #60cc60;
  border-width: 1px 1px 1px;
  border-style: solid;
  border-color:   #cecece #bababa #a8a8a8;
  border-radius: 13px 13px 12px 12px;
}

QPushButton::hover
{
  background: #50dc50;
  border-top-color: #c9c9c9;
}

QPushButton::pressed
{
  vertical-align: -5px;
  background: #319931;
  margin-bottom: 0;
  padding: 1px 13px 0;
  border-width: 0;
  border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
   <property name="shortcut">
    <string>Enter</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono NL</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
  margin: 0 0 5px;
  padding: 0 12px;
  font-size: 16px;
  font-weight: bold;
  color: #eeeeee;
  background: #c24242;
  border-width: 1px 1px 1px;
  border-style: solid;
  border-color:   #cecece #bababa #a8a8a8;
  border-radius: 13px 13px 12px 12px;
}

QPushButton::hover
{
  background: #cc6060;
  border-top-color: #c9c9c9;
}

QPushButton::pressed
{
  vertical-align: -5px;
  background: #993131;
  margin-bottom: 0;
  padding: 1px 13px 0;
  border-width: 0;
  border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_del">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono NL</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>del</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="x_value">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono NL</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
